pkg/vpcmodel/output.go

```diff
@@ -106,6 +106,7 @@ func NewOutputGenerator(c1, c2 map[string]*VPCConfig, grouping bool, uc OutputUs
 			}
 		}
 	}
+	unifyMultiVPC(c1, res.NodesConn, res.SubnetsConn)
```

Comment:
do we need unifyMultiVPC() in archMode?

Reply:
Probably not - as there is nothing to group in `archMode`.